dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -1105,11 +1114,40 @@ public void swapInServerXMLFromPublish(String fileName) throws Exception {
         publishServerXML.copyToDest(serverXML, false, true);
     }
 
+    /*
+     * This method and its getter will change the validate apps setting for all servers, even ones running on other tests
+     * use setValidateAppsLocal to configure the setting for just the current server.
+     */
+    @Deprecated
     public static void setValidateApps(boolean validateApp) {
```

Comment:
We should make this proper Javadoc and direct users to the correct method. Similarly for `getValidateApps()`

```suggestion
    /**
     * Sets the global flag to control whether to check that deployed apps started successfully when the server starts.
     * @param validateApp the value to set the flag to
     * 
     * @deprecated use {@link #setValidateAppsLocal(boolean)} instead
     */
    @Deprecated
    public static void setValidateApps(boolean validateApp) {
```